src/Features/Core/Portable/PopulateSwitch/AbstractPopulateSwitchCodeFixProvider.cs

```diff
@@ -146,7 +146,7 @@ public sealed override Task RegisterCodeFixesAsync(CodeFixContext context)
                 return;
 
             var model = await document.RequireSemanticModelAsync(cancellationToken).ConfigureAwait(false);
```

Comment:
IDE change for us: we should rename RequireSemanticModelAsync to GetRequiredSemanticModel since that matches everything else we have doing this pattern.

Reply:
Yeah I found that really confusing. 

---
In reply to: [360012525](https://github.com/dotnet/roslyn/pull/40493#discussion_r360012525) [](ancestors = 360012525)